UI/MeterBrowseWnd.cpp

```diff
@@ -354,12 +358,36 @@ void MeterBrowseWnd::UpdateEffectLabelsAndValues(GG::Y& top) {
             if (const auto& building = std::dynamic_pointer_cast<const Building>(source))
                 if (const auto& planet = Objects().get<Planet>(building->PlanetID()))
                     name = planet->Name();
-            const std::string& label_template = (info.custom_label.empty()
-                ? UserString("TT_BUILDING")
-                : UserString(info.custom_label));
-            text += boost::io::str(FlexibleFormat(label_template)
-                % name
-                % UserString(info.specific_cause));
+            // same effect from several planets?
+            auto next = it + 1;
+            if (next != maybe_info_vec->end() and
```

Comment:
use the standard `&&` and `!` spellings, consistent with (as far as I know) every other use in the codebase...